.github/scripts/curate_autogenerated_schema.py

```diff
@@ -0,0 +1,1999 @@
+    schema_dict = deepcopy(schema_dict)
+
+    # Confirm definitions key
+    assert_definitions_key(schema_dict)
+
+    # Add in the $import and $include to the definitions
+    schema_dict["definitions"].update(
+        {
+            "CWLImportManual": {
+                "description": ""
+                               "Represents an '$import' directive that should point toward another compatible "
+                               "CWL file to import where specified. The contents of the imported file should be "
+                               "relevant contextually where it is being imported",
+                "$comment": ""
+                            "The schema validation of the CWL will not itself perform the '$import' to resolve and "
+                            "validate its contents. Therefore, the complete schema will not be validated entirely, "
+                            "and could still be partially malformed. "
+                            "To ensure proper and exhaustive validation of a CWL definition with this schema, "
+                            "all '$import' directives should be resolved and extended beforehand",
+                "type": "object",
+                "properties": {
+                    "$import": {
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "$import"
+                ],
+                "additionalProperties": False
+            },
+            "CWLIncludeManual": {
+                "description": ""
+                               "Represents an '$include' directive that should point toward another compatible "
+                               "CWL file to import where specified. The contents of the imported file should be "
+                               "relevant contextually where it is being imported",
+                "$comment": ""
+                            "The schema validation of the CWL will not itself perform the '$include' to resolve and "
+                            "validate its contents. Therefore, the complete schema will not be validated entirely, "
+                            "and could still be partially malformed. "
+                            "To ensure proper and exhaustive validation of a CWL definition with this schema, "
+                            "all '$include' directives should be resolved and extended beforehand",
+                "type": "object",
+                "properties": {
+                    "$include": {
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "$include"
+                ],
+                "additionalProperties": False
+            }
+        }
+    )
+
+    return schema_dict
+
+
+def fix_inline_javascript_requirement(schema_dict: Dict) -> Dict:
+    """
+    Fix the InlineJavascriptRequirement.expressionLib array to allow for $include
```

Comment:
Shouldn't this be more universal, given all the places a `$include` can occur?

Reply:
What are all the places an include can occur? 

Can an $include occur in any requirement?

Reply:
`$include` can be a key name at any level in a CWL document

https://www.commonwl.org/v1.2/Workflow.html#Document_preprocessing
https://www.commonwl.org/v1.2/SchemaSalad.html#Include

Reply:
Does this also apply for '$import'?

Reply:
If so, I'll just add them to the patternProperties of each of the definitions

Reply:
I've added $include and $import availabilities to all requirements (and hints), including top level of requirements 

So https://github.com/common-workflow-language/cwl-v1.2/blob/b60a42e3cc417c5b75b88fd7c6681abcc7ff5b89/tests/schemadef-wf.cwl#L7 would pass

as would https://github.com/common-workflow-language/cwl-v1.2/blob/b60a42e3cc417c5b75b88fd7c6681abcc7ff5b89/tests/schemadef_types_with_import-tool.cwl#L9